app/src/theme/index.tsx

```diff
@@ -387,33 +387,71 @@ function themeAggregator(darkMode: boolean) {
     ...colors(darkMode),
   }
 }
+
 export function ThemeProvider({ children }: { children: React.ReactNode }) {
   //this is temporary until we determine to use dark theme
   const darkMode = false
 
   return <StyledThemeProvider theme={themeAggregator(darkMode)}>{children}</StyledThemeProvider>
 }
 
-const TextWrapper = styled.div<{ color: keyof Colors }>`
-  color: ${props => props.theme.text3};
+const TextWrapper = styled.div<{ color: keyof Colors; letterSpacing: string }>`
+  color: ${({ color, theme }) => (theme as any)[color]};
   font-family: Roboto;
-  letter-spacing: 0.2px;
+  letter-spacing: ${props => (props.letterSpacing ? props.letterSpacing : '0.2px')};
```

Comment:
I think there is no need for letterSpacing attribute here....With new additions there seems no reason...

Reply:
Rightly yes